### ISSUE TITLE
Feature engineering

### ISSUE THREAD

**Issue by florianock** [TinkaRombouts/Titanic-Forests#2]
Hier staan ideeën voor goede feature engineering:
https://github.com/wehrley/wehrley.github.io/blob/master/SOUPTONUTS.md
- Families
  - Identificeer families voor passagiers obv achternaam en SibSp en Parch variabelen
  - Herken families die incompleet zijn in de training set. Ontbrekende familieleden zullen in de test set zitten en waarschijnlijk hetzelfde lot hebben ondergaan als hun familieleden, tenzij ze vrouw of baby waren oid.
  - Ontbrekende Age waarden kun je schatten obv Title en obv SibSp en Parch binnen een familie.
- Locaties
  - Ontbrekende cabins of decks schatten obv familie, Pclass, fare en/of Ticket
  - Oneven cabins waren aan stuurboord zijde, even aan bakboord.
- Tickets
  - Fare: delen door familie grootte waar een prijs voor een hele groep vermeld is.
- 'Women and children first' feature bepalen, age en gender geeft voorrang voor lifeboat.
